SpatialGDK/Documentation/content/get-started/gdk-and-starter-project.md

```diff
@@ -0,0 +1,70 @@
+# Get started: 3 - Set up the Unreal GDK module and Starter Project
+
+Follow the steps below to:
+
+* Clone the Starter Project repository.
+* Clone the Unreal GDK into the `Plugins` folder of your game.
+* Build the Unreal GDK module dependencies which the Starter Project needs so it can work with the GDK.
+* Run the Starter Project on SpatialOS
+
+### Cloning
```

Comment:
"Clone" rather than a gerund.

Reply:
Also what am I cloning?

Reply:
Make it match the list above...

"### Clone the Starter Project repository and the Unreak GDK repository"